src/samples/DepthMap.hpp

```diff
@@ -239,8 +239,8 @@ struct DepthMap
 	point_cloud.reserve(distances.size());
 	
 	// precompute local transformations
-	std::vector< Eigen::Transform<typename T::Scalar,3,Eigen::Affine> > rows2column;
-	std::vector< Eigen::Transform<typename T::Scalar,3,Eigen::Affine> > columns2pointcloud;
+	std::vector< Eigen::Transform<typename T::Scalar,3,Eigen::Affine>, Eigen::aligned_allocator< Eigen::Transform<typename T::Scalar,3,Eigen::Affine> > > rows2column;
```

Comment:
Disregard my comment .... just realized that it was within a method (and not as a field).